### ISSUE TITLE
Is this project still alive ?  Last released version was built in 2021

### ISSUE THREAD

**Issue by m040601** [suntong/html2md#8]
Is this project still alive ?  Last released version of this tool "suntong/html2md"was built in 2021

The library this tool is based is called  "html-to-markdown", has had and continues to have many updates and new releases in 2022 and 2023

` https://github.com/JohannesKaufmann/html-to-markdown/releases`

However the repository for this tool "suntong/html2md" hasnt been touched since 2021.

The last released version was built in 2021,
 
```
Version 0.2.2 built on 2021-12-18T21:19:32Z
Copyright (C) 2020, Tong Sun

```

There are also lots of open issues, https://github.com/suntong/html2md/issues , with no response whatsoever. Example, https://github.com/suntong/html2md/issues/4

Is anyone willing to fork this repo and provide an updated tool

**Comment by JohannesKaufmann** [comment]
Hi @m040601

it would definitely be great if @suntong finds the time to update the library. If anyone opens a _Pull Request_, I'm sure he would also appreciate that.

As for V2 of the library (currently in the works), I plan to have a Golang-API **and** CLI in the main repository.

But until then, it would be great if the existing CLI continues to work for people...

Greetings
Johannes

**Comment by suntong** [comment]
Oh, sorry @m040601, I don't know why I just notice this thread, after Johannes replied.

Sure, I was planning to do the upgrade, and will do it sooner than later.

**Comment by magnusviri** [comment]
FYI: I use this tool and am very grateful for it. It is working for me right now.

**Comment by JohannesKaufmann** [comment]
Great, nice to hear that you are back 🎉

Pinging with @ worked! I have to admit, the same happened to me. All the initial issues in the html-to-markdown repo went unnoticed because I wasn't watching my own repo 🙈

**Comment by suntong** [comment]
Hahaha, you do understand what happened to me, :) :)  
and it works great :) :)

**Comment by suntong** [comment]
Closed as completed via [f584bf7](https://github.com/suntong/html2md/commit/f584bf775faa77901bac27e1ab3416c4712544e8).

**Comment by suntong** [comment]
@all-contributors please add @magnusviri  for review,  talk, userTesting
